datasets/modeled/buildings/dimensions/regions.csv

```diff
@@ -0,0 +1,26 @@
+id,name
+TRE,Texas Reliability Entity - Texas
+SPPN,Southwest Power Pool/North - Northern Great Plains
+ISNE,NPCC/New England - New England
+PJME,PJM/East - Mid-Atlantic
+SRSE,SERC Reliability Coorporation/Southeast - Southeast
+RMRG,WECC/Rockies - Rockies
+CASO,WECC/CA South - Southern California
+MISS,Midcontinent ISO/South - Mississippi Delta
+NYCW,NPCC/NYC & Long Island - Metropolitan New York
+FRCC,Florida Reliability Coordinating Council - Florida
+NYUP,NPCC/Upstate NY - Upstate New York
+NWPP,WECC/Northwest Power Pool - Northwest
+SPPS,Southwest Power Pool/South - Southern Great Plains
+SRSG,WECC/Southwest - Southwest
+MISW,Midcontinent ISO/West - Upper Mississippi Valley
+PJMW,PJM/West - Ohio Valley
+PJMD,PJM/Dominion - Virginia
+MISC,Midcontinent ISO/Central - Middle Mississippi Valley
+SRCA,SERC Reliability Coorporation/East - Carolinas
+SPPC,Southwest Power Pool/Central - Central Great Plains
+BASN,WECC/Basin - Great Basin
+SRCE,SERC Reliability Coorporation/Central - Tennessee Valley
+CANO,WECC/CA North - Northern California
+MISE,Midcontinent ISO/East - Michigan
+PJMC,PJM/Commonwealth Edison - Metropolitan Chicago
```

Comment:
@daniel-thom are the slashes and dashes I added to this file going to be okay?

Reply:
No problem.